numpy/linalg/linalg.py

```diff
@@ -2203,17 +2206,26 @@ def lstsq(a, b, rcond="warn"):
 
     Returns
     -------
-    x : {(N,), (N, K)} ndarray
+    x : {(N,), (..., N, K)} ndarray
         Least-squares solution. If `b` is two-dimensional,
         the solutions are in the `K` columns of `x`.
-    residuals : {(1,), (K,), (0,)} ndarray
+    residuals : {(), (..., K,)} ndarray
         Sums of residuals; squared Euclidean 2-norm for each column in
```

Comment:
Never much liked this, it isn't that useful. There have been a number of requests for returning the covariance matrix of the fitted coefficients.